README.md

```diff
@@ -1,7 +1,25 @@
-# 1. Go Minitwit!
+
+# Go MiniTwit
+
+Store a file with the environment variables in the root directory with the name 
+`.minitwit-secrets.env`. 
```

Comment:
If we decide to remove minitwit-secrets.env and use env vars instead, remember to update the readme